### PR TITLE
Feature BF010. Movies Service

### PR DESCRIPTION
**Pull request by iramirezc**
Movies Service Finished using ngResource

### CODE REVIEW

**Review by CarlosPadilla — APPROVED** [review]

File: blimflix/client/app/core/core.config.js [hunk 1]
```diff
@@ -8,6 +8,8 @@
   angular.module('blimFlix.core').config(config);
 
   /* @ngInject */
-  function config() {
+  function config($routeProvider) {
+    $routeProvider
+      .otherwise('/all');
```
Comment (lordpixel) [inline blimflix/client/app/core/core.config.js:13]: Nice move, there.

File: blimflix/client/app/providers/movies.factory.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            try {
+              data = JSON.parse(data);
+
+              if(data && data.movies){
+                return data.movies;
+              }
+
+              return data;
+            } catch(e){
+              console.error("Oops ", e);
```
Comment (lordpixel) [inline blimflix/client/app/providers/movies.factory.js:42]: Backend returns status codes and messages, we should diplay that info in a user friendly way.

**Comment by lordpixel** [comment]
Just let a tiny comment, please fix before we can merge. Thanks!

File: blimflix/client/app/providers/movies.factory.js (hunk 2 above)
Comment (iramirezc) [reply blimflix/client/app/providers/movies.factory.js:42]: Ok, thanks, how should I let the user about the error, then? Or shall I just throw it again?

File: blimflix/client/app/core/core.config.js (hunk 1 above)
Comment (iramirezc) [reply blimflix/client/app/core/core.config.js:13]: It was CAPS idea ;)

File: blimflix/client/app/providers/movies.factory.js (hunk 2 above)
Comment (lordpixel) [reply blimflix/client/app/providers/movies.factory.js:42]: @iramirezc, As we spoke earlier, let's use Axel's modal implementation to display messages.
We can create a new task for this.

**Review by lordpixel — APPROVED** [review]

**Pull request merged**